graph/Cargo.toml

```diff
@@ -24,21 +25,25 @@ reqwest = { version = "0.11.18", features = ["json", "stream", "multipart"] }
 ethabi = "17.2"
 hex = "0.4.3"
 http = "0.2.3"
+hyper = { version = "1", features = ["full"] }
+http-body-util = "0.1"
+hyper-util = { version = "0.1", features = ["full"] }
 futures = { package = "futures", version = "0.1.31" }
 lru_time_cache = "0.11"
 graphql-parser = "0.4.0"
 humantime = "2.1.0"
 lazy_static = "1.4.0"
 num-bigint = { version = "=0.2.6", features = ["serde"] }
+num-integer = { version = "=0.1.45" }
 num-traits = "=0.2.17"
 rand = "0.8.4"
 regex = "1.5.4"
 semver = { version = "1.0.21", features = ["serde"] }
-serde = { version = "1.0.126", features = ["rc"] }
-serde_derive = "1.0.125"
-serde_json = { version = "1.0", features = ["arbitrary_precision"] }
-serde_regex = "1.1.0"
-serde_yaml = "0.9.21"
+serde = { workspace = true }
```

Comment:
unless we're using this longer form for a reason I think `serde.workspace = true` is a lot nicer.

Reply:
I personally like the `{ workspace = true }` notation better because it's more in line with the notation for setting features etc. but either is fine by me